Makefile

```diff
@@ -6,6 +6,8 @@ V8 ?= 11.4.183.23
 PCRE ?= 8.45
 ZLIB ?= 1.2.13
 LUAROCKS ?= 3.9.2
+PROXY_WASM_GO_SDK ?= v0.21.0
+PROXY_WASM_ASSEMBLYSCRIPT_SDK ?= v0.0.6
```

Comment:
Can we do like all other variables and remove the `v` from here, insert it in the URLs where the number is used?

Reply:
Good catch!